modules/ROOT/pages/installation.adoc

```diff
@@ -8,21 +8,26 @@ Make sure to match both the Spark version and the Scala version of your setup.
 Here is a compatibility table to help you choose the correct version of the connector.
 
 .Compatibility table
-[cols="1,1,3,2,2"]
+[cols="1,3,3"]
 |===
-| | Group ID | Artifact ID | Version (Spark 3.0 and 3.1) | Version (Spark 3.2+)
-
-|*Scala 2.12*
-|`org.neo4j`
-|`neo4j-connector-apache-spark_2.12`
-|`4.1.5_for_spark_3`
-|`{exact-connector-version}_for_spark_3`
-
-|*Scala 2.13*
-|`org.neo4j`
-|`neo4j-connector-apache-spark_2.13`
-|`4.1.5_for_spark_3`
-|`{exact-connector-version}_for_spark_3`
+| Spark Version | Artifact ID (*Scala 2.12*) | Artifact ID (*Scala 2.13*)
```

Comment:
I'm fine with the restructuring of the table, but I have a doubt about calling the whole package name "Artifact ID". Is there a better substitute for the `GroupID:ArtifactID:Version` combination?

Reply:
maybe let's just call it `Artifact`?

Reply:
Yeah, happy with `Artifact` 👍

Reply:
It might do. Or maybe `Artifact name`?

Reply:
I mean theoretically we could use `Maven coordinate` as well?

Reply:
Yes, although it may be overly specific. I think we can go with `Artifact`.